Named entrypoints (#13861)
cloudflare/cloudflare-docs: content/workers/wrangler/configuration.md

  - The name of the Worker.

- `entrypoint` {{<type>}}string{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}

>   - The name of the entrypoint to bind to, if not the default export.

Reviewer: ```suggestion
  - The name of the [entrypoint](content/workers/runtime-apis/bindings/service-bindings/rpc/#named-entrypoints) to bind to. If you do not provide a value, the default export of the Worker will be used.
```

>   - The name of the [entrypoint](content/workers/runtime-apis/bindings/service-bindings/rpc/#named-entrypoints) to bind to. If you do not provide a value, the default export of the Worker will be used.

Reviewer: ```suggestion
  - The name of the [entrypoint](content/workers/runtime-apis/bindings/service-bindings/rpc/#named-entrypoints) to bind to. If you do not specify an entrypoint, the default export of the Worker will be used.
```
